Gopkg.toml

```diff
@@ -72,7 +72,7 @@
 
 [[constraint]]
   name = "github.com/influxdata/go-syslog"
-  version = "1.0.1"
+  branch = "feature/rfc6587"
```

Comment:
Hey @leodido, are you going to release another version of go-syslog?

Reply:
Yup, going to release very soon a 2.0.0.

Need to bump up the major since there are breaking changes.